GetIntoTeachingApi/Models/BaseModel.cs

```diff
@@ -27,10 +38,13 @@ public BaseModel(Entity entity, ICrmService crm, IValidatorFactory vaidatorFacto
 
             MapFieldAttributesFromEntity(entity);
             MapRelationshipAttributesFromEntity(entity, crm, vaidatorFactory);
-
             NullifyInvalidFieldAttributes(vaidatorFactory);
-        }
-
+        }
+
+        #pragma warning disable 67
```

Comment:
I didn't even realise you could do this in C# 🤯 

Vague memories of seeing it in C++ some decades ago.